x-pack/plugins/apm/server/lib/transactions/charts/get_timeseries_data/transform.ts

```diff
@@ -82,7 +82,7 @@ export function getTpmBuckets(
 function getResponseTime(
   responseTimeBuckets: ESResponse['aggregations']['response_times']['buckets'] = []
 ) {
-  return responseTimeBuckets.slice(1, -1).reduce(
+  return responseTimeBuckets.slice(1).reduce(
     (acc, bucket) => {
       const { '95.0': p95, '99.0': p99 } = bucket.pct.values;
 
```

Comment:
I forget why we were slicing off the last bucket here in the first place?

Reply:
We had to do it to avoid gaps in the beginning and the end of ML bounds. And I think those bounds were always trimmed in the beginning and the end for some reason. We don't have to worry about gaps in the end (they can happen regardless) - but always having a gap in the beginning looks off.

I did however expand the number of buckets we get from ML to avoid this exact problem. So we _might_ not need this anymore...

Reply:
Remove the slice entirely.